bin/daps.in

```diff
@@ -62,6 +62,9 @@ VARLIST=(
     ADOC_SET
     ADOC_TYPE
     ASCIIDOC
+    ASSEMBLY
+    ASSEMBLY_DIR
+    ASSEMBLY_MAIN
```

Comment:
No, it would rather be

```
- DC-file(s)
- $ASSEMBLY_DIR
  - assembly.xml
  - topic.xml
  - topic1.xml
  - ...
  - images/src/<FORMAT>
```

ASEMBLY_DIR is defined in a DC-file can be freely chosen.

Reply:
Why not just continue to use the `xml/` dir as usual? That is imo a convention that makes sense. (And as I mention above -- for AsciiDoc, the question of assemblies is moot because AsciiDoctor already provides everything you need to implement assemblies.)

Reply:
You are free to use `xml`. In fact I am planning to get rid of the _requirement_ of having xml and adoc directories. They will be DAPS' defaults, but you will be able to overwrite this default in the DC file. This will aloow to have the DC files in a central directory and the respective XML sources in separate subdirectories (which may make organizing files easier for projects like SBPs or TRD).

Reply:
>which may make organizing files easier for projects like SBPs [...]

In the case of the SBPs, I think your proposal would be helpful but the basic issue is that all 70-odd `DC`+`xml/*.xml` files are part of the same directory. Something like this structure would both work already and make this measurably better:

```
- category-a/
  - DC-file1
  - DC-file2
  - DC-file3
  - xml/
    - MAIN-1.xml
    - MAIN-2.xml
    - MAIN-3.xml

- category-a/
  - DC-file4
  - DC-file5
  - DC-file6
  - xml/
    - MAIN-4.xml
    - MAIN-5.xml
    - MAIN-6.xml
```

Reply:
As you said, this already works. With what I have in mind you can use the structure you promised, but you would also have the _option_ of specifying subdirectories under category-a and category-b for the xml files.

As for reusing common files in several projects, I suggest using symbolic links in the file system rather than setting up something complicated within DAPS.

```
- PROJECT/
  - common-files/
    - LICENSE.xml
  - common images/
    - logo.svg
  - category-a/
    - DC-file1
    - DC-file2
    - DC-file3
    - xml/
      - MAIN-1.xml
      - MAIN-2.xml
      -> ../common-files/LICENSE.xml
    - images/src/
      - svg/
        - diagram.svg
        -> ../../../common-images/logo.svg
- category-b/
  - DC-file4
  - DC-file5
  - DC-file6
    - file4/
      - MAIN-4.xml
      -> ../../common-files/LICENSE.xml
    - file 5/
      - MAIN-5.xml
      -> ../../common-files/LICENSE.xml
    - file6/ 
      - MAIN-6.xml
      -> ../../common-files/LICENSE.xml
```